Dockerfile

```diff
@@ -79,10 +79,6 @@ COPY --from=builder /app/bin /app/bin
 COPY --from=builder /app/version.json /app
 COPY --from=builder /app/config /app/config
 
-ARG HOST=0.0.0.0
-ARG PORT=8080
-ENV MERINO_HTTP__LISTEN="${HOST}:${PORT}"
```

Comment:
This was the real fix. The rest of the changes in this commit are useful things I figured out while debugging this.